packages/kit/src/core/prerender/prerender.js

```diff
@@ -133,7 +119,29 @@ export async function prerender() {
 	const server = new Server(manifest);
 	await server.init({ env: JSON.parse(env) });
 
-	const error = normalise_error_handler(log, config);
+	const handle_http_error = normalise_error_handler(
+		log,
+		config.prerender.handleHttpError,
+		({ status, path, referrer, referenceType }) => {
+			const message =
+				status === 404 && !path.startsWith(config.paths.base)
+					? `${path} does not begin with \`base\`, which is configured in \`paths.base\` and can be imported from \`$app/paths\` - see https://kit.svelte.dev/docs/configuration#paths for more info`
```

Comment:
```suggestion
					? `${path} does not begin with \`${config.paths.base}\`, which is configured in \`paths.base\` and can be imported from \`$app/paths\` - see https://kit.svelte.dev/docs/configuration#paths for more info`
```

Reply:
"foo does not begin with `bar`, which is configured in `paths.base` and ..." sounds better to me than "foo does not begin with `base`, which is configured in `paths.base` and ..."

Reply:
The trick is to steer people away from seeing `"/bar/"` in the message and just hardcoding that prefix

Reply:
If you don't want to include the actual value, maybe completely rephrasing it can help transport that and steer users in the right direction? I got confused by the static `base` that doesn't make much sense when reading the error message in isolation.